packages/purpose-process/test/clonePurpose.test.ts

```diff
@@ -401,4 +625,223 @@ describe("clonePurpose", async () => {
       })
     ).rejects.toThrowError(tenantKindNotFound(mockTenant.id));
   });
+  it("should throw organizationIsNotTheDelegatedConsumer when the requester is the Consumer and is cloning a purpose created by the delegate in clonePurpose", async () => {
+    const consumer = {
+      ...getMockTenant(),
+      kind: tenantKind.PA,
+    };
+
+    const consumerDelegate = {
+      ...getMockTenant(),
+      kind: tenantKind.PA,
+    };
+
+    const mockEService = getMockEService();
+
+    const mockAgreement = getMockAgreement(
+      mockEService.id,
+      consumer.id,
+      agreementState.active
+    );
+
+    const purposeCreatedByDelegate: Purpose = {
+      ...getMockPurpose(),
+      eserviceId: mockEService.id,
+      consumerId: consumer.id,
+      versions: [getMockPurposeVersion(purposeVersionState.active)],
+      delegationId: generateId<DelegationId>(),
+    };
+
+    const delegation = getMockDelegation({
+      id: purposeCreatedByDelegate.delegationId,
+      kind: delegationKind.delegatedConsumer,
+      eserviceId: purposeCreatedByDelegate.eserviceId,
+      delegatorId: purposeCreatedByDelegate.consumerId,
+      delegateId: consumerDelegate.id,
+      state: delegationState.active,
+    });
+
+    await addOnePurpose(purposeCreatedByDelegate);
+    await addOneTenant(consumer);
+    await addOneTenant(consumerDelegate);
+    await addOneAgreement(mockAgreement);
+    await addOneDelegation(delegation);
+
+    expect(
+      purposeService.clonePurpose({
+        purposeId: purposeCreatedByDelegate.id,
+        seed: {
+          eserviceId: mockEService.id,
+        },
+        ctx: {
+          authData: getRandomAuthData(consumer.id),
+          correlationId: generateId(),
+          logger: genericLogger,
+          serviceName: "test",
+        },
+      })
+    ).rejects.toThrowError(
+      organizationIsNotTheDelegatedConsumer(consumer.id, delegation.id)
+    );
+  });
+  it("should throw organizationIsNotTheConsumer when the requester is the Consumer with no delegation and is cloning a purpose created by a delegate in clonePurpose", async () => {
+    const consumer = {
+      ...getMockTenant(),
+      kind: tenantKind.PA,
+    };
+
+    const consumerDelegate = {
+      ...getMockTenant(),
+      kind: tenantKind.PA,
+    };
+
+    const mockEService = getMockEService();
+
+    const mockAgreement = getMockAgreement(
+      mockEService.id,
+      consumer.id,
+      agreementState.active
+    );
+
+    const purposeCreatedByDelegate: Purpose = {
+      ...getMockPurpose(),
+      eserviceId: mockEService.id,
+      consumerId: consumerDelegate.id,
```

Comment:
After #1407 it should work with consumer.id here, it must be consumer.id because "requester is the Consumer"
```suggestion
      consumerId: consumer.id,
```